src/dao/ERC20DAO.js

```diff
@@ -11,6 +11,7 @@ export const TX_TRANSFER = 'transfer'
 export default class ERC20DAO extends AbstractTokenDAO {
   constructor (at) {
     super(require('chronobank-smart-contracts/build/contracts/ERC20Interface.json'), at)
+    this._txOkCodes = [true]
```

Comment:
By the way, does ERC20 standardize such boolean results? Because ERC20DAO is not only for our ERC20 contracts, but for any user ERC20 contracts